packages/vue-client/src/components/GameList.vue

```diff
@@ -1,19 +1,56 @@
 <script setup lang="ts">
 import { storeToRefs } from "pinia";
+import { ref } from "vue";
 import { useStore } from "../stores/games";
 
 const store = useStore();
 const { games } = storeToRefs(store);
 
-store.fetchGames();
+const countOptions = [10, 15, 25, 50];
+const count = ref(countOptions[0]);
+const offset = ref(0);
+
+store.fetchGames(count.value, offset.value);
+
+const fetchGames = () => store.fetchGames(count.value, offset.value);
+
+const first = () => {
+  offset.value = 0;
+  fetchGames();
```

Comment:
Doesn't have to change in this PR, but it occurred to me: we should probably rely more on Vue's reactivity to trigger the fetch when the page changes.  This would make the code a lot more concise as we wouldn't have to call `fetchGames()` explicitly in each click handler.   Something like:

```
const count = ...;
const offset = ...;

const games_url = `${PREFIX}/games?count=${count ?? 0}&offset=${offset ?? 0}`
// XHR will be sent every time the url changes, data will update when the request returns
const { data } = useFetch(games_url);
```